src/meta/proto-conv/tests/it/v028_schema.rs

```diff
@@ -34,19 +34,19 @@ use crate::common;
 #[test]
 fn test_decode_v28_schema() -> anyhow::Result<()> {
     let schema_v28 = [
-        10, 28, 10, 1, 97, 26, 17, 154, 2, 8, 34, 0, 160, 6, 27, 168, 6, 24, 160, 6, 27, 168, 6,
-        24, 160, 6, 27, 168, 6, 24, 10, 104, 10, 1, 98, 26, 91, 202, 2, 82, 10, 2, 98, 49, 10, 2,
+        10, 28, 10, 1, 97, 26, 17, 154, 2, 8, 34, 0, 160, 6, 28, 168, 6, 24, 160, 6, 28, 168, 6,
+        24, 160, 6, 28, 168, 6, 24, 10, 104, 10, 1, 98, 26, 91, 202, 2, 82, 10, 2, 98, 49, 10, 2,
         98, 50, 18, 47, 202, 2, 38, 10, 3, 98, 49, 49, 10, 3, 98, 49, 50, 18, 9, 138, 2, 0, 160, 6,
-        27, 168, 6, 24, 18, 9, 146, 2, 0, 160, 6, 27, 168, 6, 24, 160, 6, 27, 168, 6, 24, 160, 6,
-        27, 168, 6, 24, 18, 17, 154, 2, 8, 66, 0, 160, 6, 27, 168, 6, 24, 160, 6, 27, 168, 6, 24,
-        160, 6, 27, 168, 6, 24, 160, 6, 27, 168, 6, 24, 32, 1, 160, 6, 27, 168, 6, 24, 10, 30, 10,
-        1, 99, 26, 17, 154, 2, 8, 34, 0, 160, 6, 27, 168, 6, 24, 160, 6, 27, 168, 6, 24, 32, 4,
-        160, 6, 27, 168, 6, 24, 10, 49, 10, 10, 100, 101, 99, 105, 109, 97, 108, 49, 50, 56, 26,
-        27, 218, 2, 18, 10, 10, 8, 18, 16, 3, 160, 6, 27, 168, 6, 24, 160, 6, 27, 168, 6, 24, 160,
-        6, 27, 168, 6, 24, 32, 5, 160, 6, 27, 168, 6, 24, 10, 49, 10, 10, 100, 101, 99, 105, 109,
-        97, 108, 50, 53, 54, 26, 27, 218, 2, 18, 18, 10, 8, 46, 16, 6, 160, 6, 27, 168, 6, 24, 160,
-        6, 27, 168, 6, 24, 160, 6, 27, 168, 6, 24, 32, 6, 160, 6, 27, 168, 6, 24, 24, 7, 160, 6,
-        27, 168, 6, 24,
+        28, 168, 6, 24, 18, 9, 146, 2, 0, 160, 6, 28, 168, 6, 24, 160, 6, 28, 168, 6, 24, 160, 6,
+        28, 168, 6, 24, 18, 17, 154, 2, 8, 66, 0, 160, 6, 28, 168, 6, 24, 160, 6, 28, 168, 6, 24,
+        160, 6, 28, 168, 6, 24, 160, 6, 28, 168, 6, 24, 32, 1, 160, 6, 28, 168, 6, 24, 10, 30, 10,
+        1, 99, 26, 17, 154, 2, 8, 34, 0, 160, 6, 28, 168, 6, 24, 160, 6, 28, 168, 6, 24, 32, 4,
+        160, 6, 28, 168, 6, 24, 10, 49, 10, 10, 100, 101, 99, 105, 109, 97, 108, 49, 50, 56, 26,
+        27, 218, 2, 18, 10, 10, 8, 18, 16, 3, 160, 6, 28, 168, 6, 24, 160, 6, 28, 168, 6, 24, 160,
+        6, 28, 168, 6, 24, 32, 5, 160, 6, 28, 168, 6, 24, 10, 49, 10, 10, 100, 101, 99, 105, 109,
+        97, 108, 50, 53, 54, 26, 27, 218, 2, 18, 18, 10, 8, 46, 16, 6, 160, 6, 28, 168, 6, 24, 160,
+        6, 28, 168, 6, 24, 160, 6, 28, 168, 6, 24, 32, 6, 160, 6, 28, 168, 6, 24, 24, 7, 160, 6,
+        28, 168, 6, 24,
     ];
```

Comment:
Hmmm... why are the bytes changed? These data is built with older version and is meant to assert compatibility. Therefore once it is created it should not be updated.

Reply:
`0028` is not added to `main.rs`, the previous bytes are copied from `0027`, so it's not correct.